service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventTestData.java

```diff
@@ -180,13 +195,13 @@ public static SupportingEvidenceBundle  createDummyEvidenceBundle(final String n
 
     public static ApplicationDocument createDummyApplicationDocument(final String name, final String uploadedBy,
                                                                      DocumentReference docRef) {
-        return ApplicationDocument.builder()
-                .documentName(name)
-                .documentType(BIRTH_CERTIFICATE)
-                .uploadedBy(uploadedBy)
-                .document(docRef)
-                .dateTimeUploaded(LocalDateTime.now())
-                .build();
+        ApplicationDocument.ApplicationDocumentBuilder document = ApplicationDocument.builder()
+            .documentName(name)
+            .documentType(BIRTH_CERTIFICATE)
+            .uploadedBy(uploadedBy)
+            .document(docRef)
+            .dateTimeUploaded(LocalDateTime.now());
+        return document.build();
```

Comment:
I'm not sure this change is necessary? can still do the return ApplicationDocument.builder().stuff().build()